File: Module_01/7.OOP/BaiTap/BaiTapThucTe_test/Customer.js
let Customer = function () {
    this.setName = function (name) {
        this.name = name;
    };
    this.getName = function () {
        return this.name;
    };
    this.setAddress = function (address) {
        this.address = address;
    };
    this.getAddress = function () {
        return this.address
    };
};

File: Module_02/java_admin/8.Spring-Data-Repository/demo-jpa/src/main/java/com/codegym/jpa/demojpa/model/Customer.java
package com.codegym.jpa.demojpa.model;

import javax.persistence.*;

@Entity
@Table(name = "C1119G1")
public class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    private String name;

    private String address;

    public Customer() {
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }
}
